utilities/pipelines/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -3,7 +3,7 @@
 Remove a specific resource
 
 .DESCRIPTION
-Remove a specific resource. Tries to handle different resource types accordingly
+Remove a specific resource. Tries to handle different resource types accordingly.
```

Comment:
Seems random when not updated on other sentences in the script doc too.

Reply:
Removed